Rakefile

```diff
@@ -12,7 +12,7 @@ RuboCop::RakeTask.new
 require 'cucumber/rake/task'
 Cucumber::Rake::Task.new
 
-default_tasks = %i[spec cucumber cck rubocop]
```

Comment:
Is this because truffleruby won't run rubocop? Or some other reason?

Reply:
> The main reason is that currently we have memory issue with GithubActions and I noticed that running tests together with rubocop fails the pipeline, which is currently the team is investigating, but I didn't want to block this PR with that Regarding the Truffleruby support, if pipeline is failing and others no please create an issue in the Truffleruby repository and the team will help with that